plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -312,12 +317,38 @@ public static String quotedTableName(SchemaTableName name)
 
     private static String quotedName(String name)
     {
-        if (SIMPLE_NAME.matcher(name).matches()) {
+        if (UNQUOTED_IDENTIFIER_PATTERN.matcher(name).matches()) {
             return name;
         }
         return '"' + name.replace("\"", "\"\"") + '"';
     }
 
+    public static String fromColumnToIdentifier(String column)
```

Comment:
The method introduced here is unused (in the commit which introduced it), and it's unclear what's the context in which is should be used.
Squash the changes with the next commit.